plugin/goto.py

```diff
@@ -58,7 +58,8 @@ def handle_response(self, response: Any, side_by_side: bool) -> None:
         if window is None:
             return
         if response:
-            self.view.run_command("add_jump_record", {"selection": [(r.a, r.b) for r in self.view.sel()]})
+            if any(view.sel()):
```

Comment:
I don't think `any` is necessary. As long as list has some element, it's valid IMO.

Reply:
Wait, actually it can even cause bugs. We should still add jump record for empty selection (empty as in, starts and stops at the same point).

Reply:
`view.sel()` returns a `sublime.Selection` object, which has a custom `__bool__` that doesn't do what you'd expect :/ I've used `len(view.sel()) > 0` instead.

Reply:
yes, it returns False for empty selection so `any(view.sel())` would return False for empty selection (starts and ends in same point).